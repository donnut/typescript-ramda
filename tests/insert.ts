import R_insert = require('../ramda/dist/src/insert');

declare const object: object;
declare const object_array: object[];
declare const string: string;
declare const number: number;

// @dts-jest:pass:snap
R_insert(number, string);
// @dts-jest:pass:snap
R_insert(number, string, object_array);
// @dts-jest:pass:snap
R_insert(number)(string)(object_array);

// @dts-jest:pass:snap
R_insert(number, object);
// @dts-jest:pass:snap
R_insert(number, object, object_array);
// @dts-jest:pass:snap
R_insert(number)(object)(object_array);
