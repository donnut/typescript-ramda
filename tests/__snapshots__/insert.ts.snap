// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.insert(number)(object)(object_array) 1`] = `"object[]"`;

exports[`R.insert(number)(string)(object_array) 1`] = `"(string | object)[]"`;

exports[`R.insert(number, object) 1`] = `"<U>(list: U[] | ArrayLike<U>) => (object | U)[]"`;

exports[`R.insert(number, object, object_array) 1`] = `"object[]"`;

exports[`R.insert(number, string) 1`] = `"<U>(list: U[] | ArrayLike<U>) => (string | U)[]"`;

exports[`R.insert(number, string, object_array) 1`] = `"(string | object)[]"`;
